src/pages/index.html

```diff
@@ -25,52 +25,115 @@
-              <span class="button button--duw">
+
+            <div class="text--small">Lorem</div>
+
+            <div class="download-list__item">
+              <div class="download__file">
                 <span>Lorem</span>
                 <i data-feather="download-cloud"></i>
-              </span>
+              </div>
+              <span class="download__date">12.31.23</span>
             </div>
-            <span class="date">123 123</span>
-          </div>
-          <div class="foo">
-            <div class="box-button">
-              <span class="button button--duw">
+
+            <div class="download-list__item">
+              <div class="download__file">
                 <span>Lorem</span>
                 <i data-feather="download-cloud"></i>
-              </span>
+              </div>
+              <span class="download__date">12.31.23</span>
             </div>
-            <span class="date">123 123</span>
-          </div>
-          <div class="foo">
-            <div class="box-button">
-              <span class="button button--duw">
+
+            <div class="download-list__item">
+              <div class="download__file">
                 <span>Lorem</span>
                 <i data-feather="download-cloud"></i>
-              </span>
+              </div>
+              <span class="download__date">12.31.23</span>
             </div>
-            <span class="date">123 123</span>
+
+            <div class="download-list__item">
+              <div class="download__file">
+                <span>Lorem</span>
+                <i data-feather="download-cloud"></i>
+              </div>
+              <span class="download__date">12.31.23</span>
+            </div>
+
+            <div class="download-list__item">
+              <div class="download__file">
+                <span>Lorem</span>
+                <i data-feather="download-cloud"></i>
+              </div>
+              <span class="download__date">12.31.23</span>
+            </div>
+
+            <div class="download-list__item">
+              <div class="download__file">
+                <span>Lorem</span>
+                <i data-feather="download-cloud"></i>
+              </div>
+              <span class="download__date">12.31.23</span>
+            </div>
+
           </div>
+
+          <span>Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptas qui perferendis dicta ratione! Assumenda.</span>
+
         </div>
+
+        <div class="card card__content">
+          <div class="stats">
+
+            <div class="stats__item">
+              <div class="text--lead">123 456</div>
+              <div class="text--smallest">lorem </div>
+            </div>
+
+            <div class="stats__item">
+              <div class="text--lead">458</div>
+              <div class="text--smallest">lorem </div>
+            </div>
+
+            <div class="stats__item">
+              <div class="text--lead">123 45</div>
```

Comment:
It is not very semantic to put text into component like `div`.

`div` is used primarily for having another elements inside. Please put text into text elements like `p`, `span`, etc.